README.md

```diff
@@ -155,6 +155,22 @@ For instance, you can ignore all the npm modules:
 ignoreFilenames: ['node_modules'],
 ```
 
+### `additionalLibraries`
+
+This option gives the possibility to remove other `propTypes` in addition to the canonical `prop-types`.
+
+For instance, by default
+```js
+import PropTypes from 'prop-types'
+import ImmutablePropTypes from 'react-immutable-proptypes'
+```
+will result in `propTypes` not to be removed; with:
```

Comment:
uh, I (probably) sorted it out! It doesn't behaves this way because of the lack of #103: I use both `'prop-types'`, `'react-immutable-proptypes'`, and `connect` from `'react-redux'`, so nothing is removed due to the latter.